fuel-p2p/Cargo.toml

```diff
@@ -13,4 +13,6 @@ log = "0.4.8"
 sha2 = "0.9"
 
 [dev-dependencies]
+env_logger = "0.9"
 rand = "0.8.4"
+tokio = {version = "1.8", features = ["full"]}
```

Comment:
```suggestion
tokio = { version = "1.8", features = ["full"] }
```

Reply:
ah so my toml formatter removes that extra space between - but I see others are not removing it - so I need to comply :)